cluster-operator/src/main/java/io/strimzi/operator/cluster/model/ZookeeperCluster.java

```diff
@@ -409,7 +409,12 @@ public StrimziPodSet generatePodSet(int replicas,
                 ownerReference,
                 templatePodSet,
                 replicas,
-                Map.of(Annotations.ANNO_STRIMZI_IO_STORAGE, ModelUtils.encodeStorageToJson(storage)),
+                Map.of(
+                        Annotations.ANNO_STRIMZI_IO_STORAGE, ModelUtils.encodeStorageToJson(storage),
+                        //Take ownership of the rolling update annotation (for SSA) and reset it
+                        //as (potential) one-time roll was already ran by reconcile loop run if it was needed.
+                        Annotations.ANNO_STRIMZI_IO_MANUAL_ROLLING_UPDATE, "false"
```

Comment:
We first need to take control of it by setting any value and then setting null would you remove the annotation.
Since we are not owners of the annotation setting null (so not including annotation in the request) would be just noop.

I went with just setting "false" here but we could implement logic to set null / remove annotation during subsequent loops if it's false.